tests/project.go

```diff
@@ -43,3 +43,92 @@ func TestCreateProject(t *testing.T) {
 	}
 	assert.True(t, sawNewProject)
 }
+
+func TestUpdateProjectDescription(t *testing.T) {
+	ctx := context.Background()
+	client, conn := GetTestAdminServiceClient()
+	defer conn.Close()
+
+	// Create a new project.
+	req := admin.ProjectRegisterRequest{
+		Project: &admin.Project{
+			Id:   "potato",
+			Name: "spud",
```

Comment:
nit: for completeness, want to add Labels here so we make sure that emtpy labels in the update request below doesn't delete them?

Reply:
Yup, added [here](https://github.com/lyft/flyteadmin/pull/114/files#r471760725).